bedrock/settings/static_media.py

```diff
@@ -358,12 +358,11 @@
         ),
         'output_filename': 'css/nightly_firstrun-bundle.css',
     },
-    'firefox_variants_firstrun': {
+    'firefox_firstrun': {
         'source_filenames': (
             'css/sandstone/sandstone.less',
             'css/tabzilla/tabzilla-static.less',
-            'css/firefox/australis/fx40/firstrun.less',
-            'css/firefox/firstrun/variants.less',
+            'css/firefox/firstrun/firstrun.less',
         ),
         'output_filename': 'css/firefox_fx40_firstrun-bundle.css',
```

Comment:
I failed to catch this in my review of the variants last week, but this output bundle name is the same as line 346 above. If this new firstrun bundle is fully replacing the fx40 firstrun styles then we should delete the previous block in static_media, but if we need to keep the older fx40 styles and these are a new set for newer versions, the output bundle name should be different.